diff -ruN a/dots/.config/quickshell/ii/modules/ii/cheatsheet/CheatsheetKeybindsCategory.qml b/dots/.config/quickshell/ii/modules/ii/cheatsheet/CheatsheetKeybindsCategory.qml
--- a/dots/.config/quickshell/ii/modules/ii/cheatsheet/CheatsheetKeybindsCategory.qml	2026-09-22 22:32:25.931508867 +0200
+++ b/dots/.config/quickshell/ii/modules/ii/cheatsheet/CheatsheetKeybindsCategory.qml	2026-09-22 22:32:31.907581286 +0200
@@ -119,7 +119,7 @@
         const key = bind.key;
         if (key.includes("mouse") || key.includes("page")) return false;
         // Contains non-1 number
-        if (/\d/.test(key) && !key.includes("1")) return true;
+        if (/^\d+$/.test(key) && key !== "1") return true;
         // Contains non-left direction
         if (/^(right|up|down)\b/i.test(key)) return true;
         return false;
@@ -132,7 +132,7 @@
 
     function transformKey(key) {
         const replaced = root.keySubstitutions[key] || key;
-        const denumbered = replaced.replace("1", "<Number>");
+        const denumbered = replaced === "1" ? "<Number>" : replaced;
         const dedirectioned = denumbered.replace("Left", "<Direction>");
         return dedirectioned;
     }
